src/shared_modules/utils/flatbuffers/schemas/syscollector_deltas.fbs

```diff
@@ -27,8 +27,6 @@ table dbsync_network_address {
     broadcast:string;
     checksum:string;
     item_id:string;
-    metric:string;
```

Comment:
Maybe this is out of the scope of the issue.
Also, it isn't recommended to remove fields from the schema
https://flatbuffers.dev/evolution/

Reply:
Thanks, is inside of this scope, but I mark them as deprecated